fake-ipa/Dockerfile

```diff
@@ -0,0 +1,9 @@
+ARG BASE_IMAGE=python:3.9-slim
+FROM $BASE_IMAGE
+ENV PBR_VERSION=1.0.0
```

Comment:
this is quite an old version, and I see that setup requires a version higher than 2.0
why setting this here?